src/v3/src/transformer/messages/transform.ts

```diff
@@ -39,7 +39,10 @@ export const CUSTOM_MESSAGE_KEYS = [
   OV_UV_ENABLE_BIOMETRIC_SERVER_KEY,
 ];
 
-const EXCLUDE_MESSAGE_STEPS = [IDX_STEP.REENROLL_AUTHENTICATOR_WARNING];
+const EXCLUDE_MESSAGE_STEPS = [
+  IDX_STEP.REENROLL_AUTHENTICATOR_WARNING,
+  IDX_STEP.REQUEST_ACTIVATION,
+];
```

Comment:
Had to exclude the `REQUEST_ACTIVATION` message from here and add it directly to the custom transformer so the title could be displayed on the top.